### DISCUSSION TITLE
osu!lazer not opening

### DISCUSSION THREAD

**Discussion by Mamapoo23583** [ppy/osu#22977] [Q&A]
### Type

Crash to desktop

### Bug description

Whenever i try to open it it just won't open.

### Screenshots or videos

![image](https://user-images.githubusercontent.com/84303356/227792524-87dd5134-9aa4-4f19-947c-bf29b11c4350.png)

### Version

2023.326.0

### Logs

(Can't find the file)

**Comment by bdach** [comment]
This report is useless without the logs. Please do try and find them.
